### ISSUE TITLE
FRames move but return to position on reload

### ISSUE THREAD

**Issue by hardvark** [GurliGebis/WoWAddon-PokemonTrainerNG#9]
When I move the frames by using the resize option, which I need to do because the frames for my team are way off to the left, they move to symmetrical positions but only until I log out. Next log in they are back where they were ie the positions are not stored.

**Comment by GurliGebis** [comment]
Hey,

Just to be sure, you are using the "More settings" button on the settings dialog to move the frames around?
I just tested that, and it seems to be working - so I'm wondering what might going on.

**Comment by hardvark** [comment]
Yes I am. I reinstalled also. It is the player team that won’t retain position. It is always far left. 


Sent from Yahoo Mail for iPhone


On Thursday, September 26, 2024, 6:28 pm, GurliGebis ***@***.***> wrote:




Hey,

Just to be sure, you are using the "More settings" button on the settings dialog to move the frames around?
I just tested that, and it seems to be working - so I'm wondering what might going on.

**Comment by GurliGebis** [comment]
Okay, would it be possible for you to record a video of what is happening? (Also, make sure all other addons are unloaded, just to be sure nothing else is interfering)
